src/components/ShowMoreButton.jsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { useState } from "react";
+import data from "../data/data";
+import CommentsSection from "./CommentsSection.jsx";
+
+function ShowMoreButton({ props, addNewNote }) {
+  const [state, setState] = useState("Show More...");
+
+  function handleSetState() {
+    state === "Show Less..."
+      ? setState("Show More...")
+      : setState("Show Less...");
+  }
+
+  const falseX = <span>❌</span>;
+  const trueCheck = <span>✅</span>;
+
+  const info = data.filter((e) => {
+    return e.id === props;
+  })[0];
+
+  function findComment(data) {
+    if (data.notes.length === 0) {
+      return <li id="no-comment">no comments added yet..</li>;
+    } else {
+      return (
+        <li id="added-comment">
+          {data.notes[0].commenter} Said: "{data.notes[0].comment}"
+        </li>
+      );
+    }
+  }
+
+  let goalPercent = (
+    (info.codewars.current.total / info.codewars.goal.total) *
+    100
+  ).toFixed(2);
+
+  return (
+    <div>
+      <button id="show-more-btn" onClick={handleSetState}>
+        {state}
+      </button>
+      {state === "Show Less..." ? (
+        <div className="show-more-info">
+          <ul>
+            <li id="codewars">
+              <h4>CodeWars</h4>
+              <hr />
+              <h5>
+                Total Current Score:{" "}
+                <span id="tally">{info.codewars.current.total}</span>
+              </h5>
+              <h5>
+                Last Week Score:{" "}
+                <span id="tally">{info.codewars.current.lastWeek}</span>
+              </h5>
+              <h5>
+                Goal:<span id="tally">{info.codewars.goal.total}</span>{" "}
+              </h5>
+
+              <div id="tally-goalPercent">
+                {goalPercent >= 100 && (
+                  <h5>
+                    <span>Percent of goal achieved: </span>
+                    <span style={{ color: "green" }}>{goalPercent}%</span>
+                  </h5>
+                )}
+                {goalPercent > 50 && goalPercent < 100 && (
+                  <h5>
+                    <span>Percent of goal achieved: </span>
+                    <span className="yellow">{goalPercent}%</span>
```

Comment:
You have the className for yellow but forgot the style attribute like you have on green and red